apps/examples/testcase/le_tc/drivers/drivers_tc_main.c

```diff
@@ -43,7 +43,7 @@ int tc_drivers_main(int argc, char *argv[])
 #endif
 
 #ifdef CONFIG_TC_DRIVERS_RTC
-	rtc_main();
+	rtc_driver_main();
```

Comment:
It is required because when we want to enable driver tc and rtc sample app simultaneously, then it will throw build error due to same api name ```rtc_main```
https://github.com/Samsung/TizenRT/blob/2901b73b196c9b49e9970bd4a76db6ae69788deb/apps/examples/rtc/rtc_main.c#L164